src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -943,27 +943,32 @@ private void ReportNonSafetyDiagnostic(ErrorCode errorCode, SyntaxNode syntax)
             Debug.Assert(!ErrorFacts.NullableFlowAnalysisSafetyWarnings.Contains(MessageProvider.Instance.GetIdForErrorCode((int)errorCode)));
             Debug.Assert(ErrorFacts.NullableFlowAnalysisNonSafetyWarnings.Contains(MessageProvider.Instance.GetIdForErrorCode((int)errorCode)));
 #pragma warning disable CS0618
-            ReportDiagnostic(errorCode, syntax);
+            ReportDiagnostic(errorCode, syntax.GetLocation());
```

Comment:
>.GetLocation() [](start = 46, length = 14)

Why change this to use the new overload?

Reply:
Because the new overload is the only overload.

---
In reply to: [267976079](https://github.com/dotnet/roslyn/pull/34273#discussion_r267976079) [](ancestors = 267976079)